testing/sdk_networks.py

```diff
@@ -1,10 +1,18 @@
+import json
+
 import shakedown
 
 
 def check_task_network(task_name, expected_network_name="dcos"):
     """Tests whether a task (and it's parent pod) is on a given network
     """
     _task = shakedown.get_task(task_id=task_name, completed=False)
+
+    if type(_task) == list:
```

Comment:
So this is a little curious to me as well. In the past this function returned a Task instance, now it returns a list of length 1 with that task. The list should always be length 1 in this case, because we're querying by task ID (which are unique). So I have the check for backwards compatibility. I like your assert(len(..)) == 1 idea though.